### ISSUE TITLE
JPEGDecompressSurface : Picture decode failed: e00002d1

### ISSUE THREAD

**Issue by Z-Kevin** [rs/SDWebImage#1839]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by mythodeia** [comment]
please share the image url so we can test it
did you try the url with the latest version of the library?

**Comment by mythodeia** [comment]
closing this. please reopen with more info

**Comment by inigo333** [comment]
This keeps happening to me. What info do you need?

**Comment by skyline75489** [comment]
@inigo333 Can you give us an image that can reproduce this error?

**Comment by Voldemarus** [comment]
On my mind it happens **while** big image is loaded and **downloadImageWithURL:** method is called. Message is generated before completion block is invoked (error = nil, finished = NO.

**Comment by O-mkar** [comment]
It happens with this URL

http://gogoanow.com/wp-content/uploads/2018/02/The-Ganesha-series-–-a-solo-exhibition-by-Hesham-Malik-e1519806035827.jpg

**Comment by mythodeia** [comment]
@O-mkar did you try this url with the latest version 4.3.2?
can you try the latest demo project with this url and report back?

**Comment by O-mkar** [comment]
Sure i will do that
